common/app-starters-micrometer-common/src/main/java/org/springframework/cloud/stream/app/micrometer/common/SpringCloudStreamMicrometerCommonTags.java

```diff
@@ -28,21 +29,21 @@
  * instance index and guids. Later are necessary to allow discrimination and aggregation of app metrics by external
  * metrics collection and visualizaiton tools.
  *
+ * Use the spring.cloud.stream.app.metrics.common.tags.enabled=false property to disable inserting those tags.
+ *
  * @author Christian Tzolov
  */
 @Configuration
+@ConditionalOnProperty(name = "spring.cloud.stream.app.metrics.common.tags.enabled", havingValue = "true", matchIfMissing = true)
 public class SpringCloudStreamMicrometerCommonTags {
 
-	@Value("${spring.cloud.dataflow.cluster.name:default}")
-	private String clusterName;
-
 	@Value("${spring.cloud.dataflow.stream.name:unknown}")
 	private String streamName;
 
 	@Value("${spring.cloud.dataflow.stream.app.label:unknown}")
```

Comment:
should this default to the normal boot application name value

Reply:
Do you mean the `spring.cloud.dataflow.stream.app.label` prop?

Reply:
The app label is the unique app name in the DSL definition. If not mistaken it defaults to the registered app name or to the label name when `label_name: app` are used in the DSL.

Reply:
ah, yes.